synapse/handlers/space_summary.py

```diff
@@ -111,28 +116,75 @@ async def get_space_summary(
             max_children = max_rooms_per_space if processed_rooms else None
 
             if is_in_room:
-                rooms, events = await self._summarize_local_room(
-                    requester, room_id, suggested_only, max_children
+                room, events = await self._summarize_local_room(
+                    requester, None, room_id, suggested_only, max_children
                 )
+
+                logger.debug(
+                    "Query of local room %s returned events %s",
+                    room_id,
+                    ["%s->%s" % (ev["room_id"], ev["state_key"]) for ev in events],
+                )
```

Comment:
Do we still want this debug logging? (I just note we're creating a list here even when debugging is off)

Reply:
I'm not sure, this was originally added by @richvdh but does seem generally helpful. This could be a rather expensive call though I suppose. My inclination would be to leave it for now until we're sure this all works? (Or maybe guard it in if debug is enabled if you'd prefer?)

Reply:
Eh, its probably fine for now